### PR TITLE
add chapter five

### PR DESCRIPTION
**Pull request by GillWu**
merge to main-comphys-112

### CODE REVIEW

File: Chapters/chapter5.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+「奶奶，」小紅帽抱怨道，「我們真的要這樣對待他嗎？或許我們可以讓他去森林裡，讓他學習如何做個好野獸。」
+
+奶奶深深地看著小紅帽，微微一笑。「孩子，」奶奶說道，「你已經展現了仁慈與智慧。或許我們可以給大野狼一個機會，看看他是否能改變。」
+
+於是，他們將大野狼解救出來，並告訴他他們的決定。大野狼眼中閃過一絲感激和驚訝，他慢慢地向小紅帽和奶奶道了謝，承諾要好好改變自己。
+
+然而，大野狼並沒有如約改變。在幾個月後，他再次出現在小紅帽的家門口，試圖以各種詭計來獵取他們的生命。
+
+這一次，小紅帽與她的奶奶準備好了。他們巧妙地利用森林中的陷阱和幻術，將大野狼永遠地困在了一個無法逃脫的陷阱中。
```
Comment (SyYunP) [inline Chapters/chapter5.md:13]: 毒蘋果 is the way

**Review by YiFang-Li — APPROVED** [review]
U should use 毒蘋果 to kill the 大野狼

**Review by Noctis0522 — APPROVED** [review]
LGTM

**Comment by Udongein3428** [comment]
Screenshot_20230713-014334_Chrome

**Review by yoyochue — APPROVED** [review]
perfect ending!!!

**Review by LiangTing-Yi — APPROVED** [review]
濕了 眼角

**Review by kuochuanpan — APPROVED** [review]
LGTM

**Review by Noctis0522 — APPROVED** [review]

**Pull request merged**